ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -2033,10 +2033,14 @@ private static void LoadAutoFilterColumns(AutoFilter af, XLAutoFilter autoFilter
                     autoFilter.Filters.Add((int)filterColumn.ColumnId.Value + 1, filterList);
 
                     Boolean isText = false;
+
+                    var formatProvider = CultureInfo.InvariantCulture.Clone() as CultureInfo;
+                    formatProvider.NumberFormat.NumberGroupSeparator = " ";
```

Comment:
I'm not convinced by this. Shouldn't we be using the user's own culture instead of forcefully specifying the number group separator? In the past I have dealt with Excel files that, when opened in Excel, will treat strings differently based on the current locale of the user. I suspect this is the case here.

Reply:
I thought so too but after I set my local settings to use a comma as a thousand separator I still got `10 000.00` in the result file.

Reply:
You must reapply the filter in Excel, then you'll see the different handling.